broker/Gemfile

```diff
@@ -65,7 +65,7 @@ end
 # Load extra gems specified in the broker configuration file
```

Comment:
Also, in the rescue block print the error to $stderr.  Also change the `puts` in line 77 to `$stderr.puts`.  That will print out the error on `service openshift-broker restart`.  I have no idea where stdout is going.